tests/TestRunner.Core/TestInstrumentation.cs

```diff
@@ -31,7 +31,7 @@ protected sealed class KnownArguments
 		Bundle arguments;
 
 		protected abstract string LogTag { get; set; }
-		protected string TestAssembliesGlobPattern { get; set; }
+		protected string TestsDirectory { get; set; }
```

Comment:
Don't assume the assemblies will actually be in a directory. With AndroidThings everything we might have to read assemblies directly from the apk - thus the glob pattern would have to become a regex then, probably.